docs/devel/developer-guides/vagrant.md

```diff
@@ -34,45 +34,64 @@ Documentation for other releases can be found at
 
 ## Getting started with Vagrant
 
-Running kubernetes with Vagrant (and VirtualBox) is an easy way to run/test/develop on your local machine (Linux, Mac OS X).
+Running kubernetes with Vagrant is a work in progress.
```

Comment:
I don't think this is appropriate.  It should work, and things that don't work are bugs.  Testing from HEAD is subject to all the same problems in all of the start-up scripts.  I am fine to say 'Running Kubernetes with Vagrant is an easy way to run/test/develop on your local machine in an environment using the same setup procedures when running on GCE or AWS cloud providers.  This provider is not tested on a per PR basis, if you experience bugs when testing from HEAD, please open an issue."